app/controllers/api/products_controller.rb

```diff
@@ -70,7 +70,7 @@ def bulk_products
 
     def overridable
       producers = OpenFoodNetwork::Permissions.new(current_api_user).
-        variant_override_producers.by_name
+        variant_override_producers.by_name.select('enterprises.id')
```

Comment:
the variable and parameter name should be changed to producer_ids for correctness.